.github/workflows/publish-container-image.yaml

```diff
@@ -0,0 +1,32 @@
+name: publish-container-image
+
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - "*"
+
+jobs:
+  docker:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
```

Comment:
I'm not 100% why qemu is needed for publishing to github registry. Would you mind explaining pls?

Reply:
It's the recommended way to use Docker on GHA, as buildx (the new Docker builder) uses it in order to provide multi platform builds.